File: test/cpp/test_136.cpp
#include "../../src/cpp/code_136.cpp"
#include "cpp_deps/boilerplate.hpp"

void test(Solution& sol, const json& input, const json& output) {
    vector<int> nums = input["nums"].get<vector<int>>();
    int expected = output.get<int>();
    int result = sol.singleNumber(nums);
    CHECK_EQ(result, expected);
}

TEST_CASE("") {
    TEST("test/test_json/test_136.json");
}

File: python/tests/test_136.py
from python.src.code_136 import Solution
from python.tests.python_deps.boilerplate import BoilerTest


def input_fetcher(case):
    return case["input"]["nums"]


def test():
    test = BoilerTest(136, input_fetcher)
    solutionXOR = Solution().singleNumberXORSolution
    solutionFreqTable = Solution().singleNumberFreqTableSolution
    test.run_tests(solutionXOR)
    test.run_tests(solutionFreqTable)
